tests/unit_tests/data_steward/tools/load_archived_submission_test.py

```diff
@@ -0,0 +1,64 @@
+from unittest import TestCase, mock
+from unittest.mock import ANY
+
+from tools import load_archived_submission as ls
+import common
+
+
+class LoadVocabTest(TestCase):
```

Comment:
nit:  Class name doesn't reflect what you're testing.

Reply:
You're right, I just added this using the vocabulary test